### PR TITLE
Read from COMMCARE_CLOUD_DEFAULT_USER env var

### PR DESCRIPTION
**Pull request by dannyroberts**
fixes https://github.com/dimagi/commcare-cloud/issues/2410

### CODE REVIEW

File: src/commcare_cloud/commands/terraform/aws.py [hunk 1]
```diff
@@ -205,9 +216,11 @@ def aws_sign_in(aws_profile, duration_minutes=DEFAULT_SIGN_IN_DURATION_MINUTES,
             and _has_valid_session_credentials(aws_session_profile):
         return aws_session_profile
 
-    default_username = getpass.getuser()
+    default_username = get_default_username()
     username = input("Enter username associated with credentials [{}]: ".format(
         default_username)) or default_username
```
Comment (millerdev) [inline src/commcare_cloud/commands/terraform/aws.py]: This requires a tiny bit more user interaction than I think is necessary. Having `COMMCARE_CLOUD_DEFAULT_USERNAME` set in the environment indicates that it has been explicitly overridden. In that case it should not bother the user with this additional prompt.

File: src/commcare_cloud/commands/terraform/aws.py [hunk 2]
```diff
@@ -164,6 +164,17 @@ def run(self, args, unknown_args):
...
 
+@memoized
+def get_default_username():
+    return os.environ.get('COMMCARE_CLOUD_DEFAULT_USERNAME', getpass.getuser())
+
+
+def print_help_message_about_the_commcare_cloud_default_username_env_var():
+    puts(colored.blue("Did you know? You can put"))
+    puts(colored.blue("    export COMMCARE_CLOUD_DEFAULT_USERNAME={}".format(username)))
+    puts(colored.blue("in your profile to never have to type that in again! 🌈"))
```
Comment (pr33thi) [inline src/commcare_cloud/commands/terraform/aws.py]: this is such nice help text

**Comment by pr33thi** [comment]
Excited for this to get added to the code!

**Comment by dannyroberts** [comment]
@millerdev Okay, I implemented your suggestion to only ask for your username if it's not set in the environment variable

File: src/commcare_cloud/commands/terraform/aws.py [hunk 3]
```diff
@@ -164,12 +164,30 @@ def run(self, args, unknown_args):
 DEFAULT_SIGN_IN_DURATION_MINUTES = 30
 
 
+class StringIsGuess(unicode):
```
Comment (millerdev) [inline src/commcare_cloud/commands/terraform/aws.py]: `unicode` -> `six.text_type`

**Pull request merged**